js/product-page.js

```diff
@@ -30,7 +32,11 @@ const eror404 = function() {
     .classList.add("product-page-warning");
 };
 
-const buildProductPage = function(data) {
+const breadcrumb = function (data) {
```

Comment:
please, move this function in index.js file (create if needed)
and look at all pages - we should reuse this all the time we need breadcrumbs
we should pass throw it some parameters (think what exactly) and it should be reusable from all places